<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuffPackEdit</class>
 <widget class="QDialog" name="StuffPackEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>498</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdExit">
       <property name="text">
        <string>Beenden</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblIniFile">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="grpGlobal">
       <property name="title">
        <string>Global</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLineEdit" name="edtLogo"/>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="edtCodeTable"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Logo Publisher:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="edtSoundtrack"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="edtLogoDeveloper"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblLogoDeveloper">
          <property name="text">
           <string>Logo Developer:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblWiki">
          <property name="text">
           <string>Wiki:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="edtWiki"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblVideo">
          <property name="text">
           <string>Video:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Codetable:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblSoundtrack">
          <property name="text">
           <string>Soundtrack</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="edtVideo"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Manual:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="edtManual"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Solution:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="edtSolution"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Informations</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Developer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtDeveloper"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Publisher:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edtPublisher"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Releasedate:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtReleasedate"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Rating:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edtRating"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Players:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edtPlayers"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Description</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QTextEdit" name="edtInfo"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCover">
      <attribute name="title">
       <string>Cover</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpCover">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtAddCoverList"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="cmdCoverAdd">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cmdCoverRemove">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="lstCovers"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMedia">
      <attribute name="title">
       <string>Media</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpMedia">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QListWidget" name="lstMedia"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtAddMediaList"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cmdMediaRemove">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="cmdMediaAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Screenshot</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpScreenshot">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtAddScreenshotList"/>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="lstScreenshot"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cmdScreenshotRemove">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="cmdScreenshotAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Soundtracks</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpSoundtrack">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtAddSoundtrackList"/>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="lstSoundtrack"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cmdSoundtrackRemove">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="cmdSoundtrackAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>WEB-Links</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpLinks">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="1" column="0">
           <widget class="QLineEdit" name="edtAddLinksList"/>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="lstLinks"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cmdLinksRemove">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="cmdLinksAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../tosec.qrc">
              <normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</normaloff>:/icon/farious/blue/images/farious/blue/pipette.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Quick Links</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QTableWidget" name="twQuickLinks">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>10</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../tosec.qrc"/>
 </resources>
 <connections/>
</ui>
